src/action/darwin/create_synthetic_objects.rs

```diff
@@ -2,19 +2,16 @@ use tokio::process::Command;
 
 use crate::execute_command;
 
-use crate::action::{Action, ActionDescription, ActionState};
+use crate::action::{Action, ActionDescription, StatefulAction};
 
+/// Create the synthetic objects defined in `/etc/syntethic.conf`
 #[derive(Debug, serde::Deserialize, serde::Serialize, Clone)]
-pub struct CreateSyntheticObjects {
-    action_state: ActionState,
-}
+pub struct CreateSyntheticObjects {}
```

Comment:
Nit, but would it make a difference if this was a unit struct, i.e.:

```suggestion
pub struct CreateSyntheticObjects;
```

?

Reply:
I think that would be fine!